tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -975,7 +1030,9 @@ def rbd_mirror_config(**kw):
         RBD mirror objects for ecpool and replicated pool
     """
     rbd_obj = dict()
-
+    log.debug(
```

Comment:
is debug log required

Reply:
IMHO, it helps in debugging and avoids initiating runs with pdb for most of the times.

Reply:
Resolving based on discussions in review meeting, please re-open if required